protocols/tck/cloudstate/tck/model/eventlogeventing.proto

```diff
@@ -0,0 +1,178 @@
+// Copyright 2019 Lightbend Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+//
+// == Cloudstate TCK model test for event log eventing ==
+//
+
+syntax = "proto3";
+
+package cloudstate.tck.model.eventlogeventing;
+
+option java_package = "io.cloudstate.tck.model";
+option go_package = "github.com/cloudstateio/go-support/tck/eventlogeventing;eventlogeventing";
+
+import "google/protobuf/any.proto";
+import "google/protobuf/empty.proto";
+import "cloudstate/eventing.proto";
+import "cloudstate/entity_key.proto";
+
+//
+// The `EventLogSubscriberModel` service is an Action that should be implemented in the following ways:
+//
+// - The `ProcessEventOne` method receives an `EventOne` message, and must behave according to the passed in step.
+// - The `ProcessEventTwo` method receives an `EventTwo` message, and must return a stream of responses, one for each
+//   incoming step.
+// - The `Effect` method receives an `EffectRequest` message and must respond with a `Response` that contains the id
+//   from the effect message.
+// - The `ProcessAnyEvent` method receives a `google.protobuf.Any`, which will contain JSON serialized according to
+//   the Cloudstate JSON serialization conventions, with a `type_url` of `json.cloudstate.io/JsonEvent`. The contents
```

Comment:
I think the mandated value of `json.cloudstate.io/JsonEvent` here can't be enforced as different languages choose "a path determined using a language-specific mechanism" for the type serialized:
As per https://cloudstate.io/docs/contribute/serialization.html#json-values, 

Although the TCK does not enforce the `type_url`described here:
https://github.com/cloudstateio/cloudstate/commit/b722f866ce962814e7b23efd3c7f69c702148c66#diff-e685c4ba3ccc54f02fab8a427f5f505c63fc8edd56b92dab716e934ccacb2b9aR3295

Reply:
The suffix can be anything - but the proxy handles JSON in a particular way, so the TCK does indirectly enforce it.

Reply:
>The suffix can be anything

Then this:
>with a `type_url` of `json.cloudstate.io/JsonEvent`

is inaccurate I think. The suffix is out of reach to be defined by the TCK model implemented. Instead it's what the language support has decided to be for a type of a languages types for the Cloudstate JSON serialisation implementation.